mmvb_backend/common/models.py

```diff
@@ -3,8 +3,9 @@
 from django.contrib.auth.models import AbstractUser
 from django.db import models
 
-# TODO: those states should be decided upon for proper
-# TODO: implementation of the reviewing flow
+# TODO: (Future) those states should be decided upon for proper
+# TODO: (Future) implementation of the reviewing flow when publishing
+# TODO: (Future) cases
```

Comment:
We have a ticket for that right? Maybe worth mentioning here 👍

Reply:
Not really, we don't have a ticket for that yet since this was only implemented that way (and hence the TODO comments) because of the vague description of a future feature of "being able to submit cases/case sets to approval to be used in benchmarks, approving or rejecting those, etc..."